recipes/vulkan-loader/all/conanfile.py

```diff
@@ -0,0 +1,102 @@
+import os
+import glob
+from conans import ConanFile, tools, CMake
+
+
+class VulkanLoaderConan(ConanFile):
+    name = "vulkan-loader"
+    description = "Vulkan Loader"
+    topics = ("conan", "vulkan", "loader")
+    homepage = "https://github.com/KhronosGroup/Vulkan-Loader"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "arch", "build_type", "compiler"
+    generators = "cmake", "cmake_find_package"
+    options = {
+        "fPIC": [True, False],
+        "with_wsi_xcb": [True, False],
+        "with_wsi_xlib": [True, False],
+        "with_wsi_wayland": [True, False],
+        "with_wsi_directfb": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "with_wsi_xcb": False,
+        "with_wsi_xlib": False,
+        "with_wsi_wayland": False,
+        "with_wsi_directfb": False,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+        if self.options.with_wsi_wayland:
```

Comment:
We had a similar discussion in the past

In my opinion these are hidden system requirements, pretty much the lazy way and has the danger to remain a TODO forever...

but yeah, please add a least a ``# TODO line` above both warnings

However, I'm not sure if we should a allow this at all. In other recipes we rather throw exceptions too

Reply:
I think we should give the options available, but not enabled by default, while printing a warning.